assets/data/aspects/drauvenSkillDeck.json

```diff
@@ -1,37 +1,164 @@
+	"id": "drauvenDeck_cower_upgrade",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_cower",
+		"abilityDeckIcon": "ignite"
+	}
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_preparedShot",
+	"effects": [ "drauven_pc_preparedShot" ],
+	"boostType": "ABILITY_CLASS",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": { "abilityDeckUsage": "hunter", "abilityDeckIcon": "ambush" }
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_preparedShot_upgrade",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_preparedShot",
+		"abilityDeckIcon": "ambush"
+	}
+},
+{
+	"modId": "wildermyth-drauven-pcs",
 	"id": "drauvenDeck_quickblade",
-	"effects": [ "drauven_quickblade|1.6|3+1d2+((self.PHYSICAL_DAMAGE_BONUS+self.POTENCY)*0.5)"],
+	"effects": [ "drauven_pc_quickblade" ],
 	"boostType": "ABILITY_CLASS",
 	"legacyAbilityChoice": true,
-    "abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
 	"info": { "abilityDeckUsage": "hunter", "abilityDeckIcon": "thornfang" }
-  },
-  {
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_quickblade_upgrade",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_quickblade",
+		"abilityDeckIcon": "thornfang"
+	}
+},
+{
+	"modId": "wildermyth-drauven-pcs",
 	"id": "drauvenDeck_specterstep",
-	"effects": ["drauven_pc_specterstep"],
+	"effects": [ "drauven_pc_specterstep" ],
 	"boostType": "ABILITY_CLASS",
 	"legacyAbilityChoice": true,
-    "abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
 	"info": { "abilityDeckUsage": "hunter", "abilityDeckIcon": "silkstepCloak" }
-  },
-
-	{
-		"id": "drauvenDeck_blastingshard",
-		"effects": ["drauven_pc_blastingshard"],
-		"boostType": "ABILITY_CLASS",
-		"legacyAbilityChoice": true,
-    "abilityRequiresOneAspectOf": [ "drauven_pc" ],
-		"info": { "abilityDeckUsage": "mystic", "abilityDeckIcon": "shardnado" }
-	},
-	{
-		"id": "drauvenDeck_blastingshard_upgrade",
-		"boostType": "ABILITY_CLASS",
-		"legacyAbilityChoice": true,
-		"info": {
-			"abilityDeckUsage": "upgrade",
-			"upgradeForAbility": "drauvenDeck_blastingshard",
-			"abilityDeckIcon": "shardnado"
-		}
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_specterstep_upgrade",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_specterstep",
+		"abilityDeckIcon": "silkstepCloak"
+	}
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_upgraded_bloodedFerocity",
```

Comment:
I've been sticking to a pattern of `drauvenDeck_baseSkill` and `drauvenDeck_baseSkill_upgrade` to try and keep things consistent - if it's not too nitpicky, let's maybe keep the names consistent (and maybe move this to live beside the base ability definition?)